projects/go-tester/src/styles.scss

```diff
@@ -1,9 +1,11 @@
 /* You can add global styles to this file, and also import other style files */
 @import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700');
 @import url('https://fonts.googleapis.com/icon?family=Material+Icons');
+@import "~@tangoe/gosheets/gosheets";
 
 @import "~@tangoe/gosheets/gosheets";
```

Comment:
Duplicate imports.

![C-658VsXoAo3ovC](https://user-images.githubusercontent.com/1075489/57471315-e5d2d980-7258-11e9-8bbc-33c8cc531db8.jpg)

Reply:
🤦‍♂ good catch and nice gif choice!